### PR TITLE
[Feature] Add exposure.histogram

### PR DESCRIPTION
**Pull request by lewisjb**
This PR adds the `exposure.histogram` module from `skimage` to PatternFlow. It is implemented entirely in tensorflow apart from the CLI script.
There is significant documentation in the README and in the code.

The API function has argument guards to ensure correct usage and nice error messages / warnings. It also has typing as specified by PEP 484 to allow nicer usage for those developers that choose to type-check.

Student number: 44348649

### CODE REVIEW

File: image/exposure_histogram/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
+# Exposure Histrogram
```
Comment (shakes76) [inline image/exposure_histogram/README.md]: Good

File: image/exposure_histogram/impl.py [hunk 2]
```diff
@@ -0,0 +1,127 @@
+# Python built-ins, required for purely nicety reasons like raising warnings
```
Comment (shakes76) [inline image/exposure_histogram/impl.py]: Block should also include authorship, license, copyright, dates and description of what the driver does.

**Comment by shakes76** [comment]
## TF Usage
Good
## Good Practice (Design/Commenting)
Good design and comments
## Algorithm 
Driver Script present
Shows Usage & Demo & Visualisation & Data usage 
Module present
## Commit Log
Meaningful commit messages 
Progressive commits used
## Documentation
Description and Comments good
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
**Request Feedback: Rename folder**
Request Description Good

**Comment by lewisjb** [comment]
Thank you for the feedback. I have renamed the folder, and added details to the block comment of impl.py

**Pull request merged**